mimic/rest/loadbalancer_api.py

```diff
@@ -26,20 +27,20 @@
 @implementer(IAPIMock, IPlugin)
 class LoadBalancerApi(object):
     """
-    Rest endpoints for mocked Load balancer api.
+    This class registers the load balancer API in the service catalog.
+    It does NOT implement individual endpoints.
```

Comment:
I added methods which were decocorated with `@app.route`, and none of them worked (just kept getting 404s).  I agree this class would route the requests to objects with endpoint implementations, but I don't think it's built to provide implementations of endpoints.  That explains why I chose the wording I did.